### PR TITLE
switch to using different versions rather than services for deploying

### PR DESCRIPTION
**Pull request by meltsufin**
It only works with Maven for now.
TODO: We need to pass in the version for Gradle.
@loosebazooka

### CODE REVIEW

File: 9-java8-compat-flex/src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
   <threadsafe>true</threadsafe>
-  <module>9-java8-compat-flex</module>
```
Comment (loosebazooka) [inline 9-java8-compat-flex/src/main/webapp/WEB-INF/appengine-web.xml:8]: does this even do anything? might get stripped

Comment (meltsufin) [reply 9-java8-compat-flex/src/main/webapp/WEB-INF/appengine-web.xml:8]: it's equivalent to service, I believe
anyway it's gone now

**Comment by meltsufin** [comment]
@loosebazooka ready for your review

**Comment by loosebazooka** [comment]
:boat:

**Pull request merged**